src/io/sampling.jl

```diff
@@ -28,14 +28,17 @@ function adjust_samples(d::Domain, spec::DataFrame, df::DataFrame)::DataFrame
     df[df.guided.==-1.0, non_depth] .= 0.0
 
     # If no seeding is to occur, set related variables to 0
-    not_seeded = (df.seed_TA .== 0) .& (df.seed_CA .== 0.0)
+    not_seeded = (df.seed_TA .== 0.0) .& (df.seed_CA .== 0.0)
     df[not_seeded, contains.(names(df), "seed_")] .= 0.0
     df[not_seeded, :a_adapt] .= 0.0
 
     # Same for fogging/shading
     not_fogged = (df.fogging .== 0) .& (df.SRM .== 0)
```

Comment:
why are the fogging and SRM variables not compared to a float zero?

Reply:
Yeah not sure what I was seeing there, I'll change it back

Reply:
I see what happened - one condition was using an integer and the other was using a float so I just made them to match.

Corrected this so they're both now compared against integers.